empyrical/utils.py

```diff
@@ -230,10 +274,12 @@ def get_utc_timestamp(dt):
 _1_bday = BDay()
 
 
+@deprecated(msg=DATAREADER_DEPRECATION_WARNING)
```

Comment:
It's only called in functions that this PR is deprecating, and since this function is for internal use only (i.e. starts with an underscore), I thought it would make sense to deprecate it. Is that not good practice?

Reply:
You could, though the particular message is seems a bit confusing for this function. If this is called from deprecated functions, then I'd also be worried about warning spam from this. Since this is a "private" function (in general, functions prefixed with an `_` are private by convention), it doesn't seem all that important to me to deprecate this.

Reply:
Removed deprecation warning.